abc.templates/infra/main.tf

```diff
@@ -0,0 +1,48 @@
+locals {
+  project_id                        = "REPLACE_PROJECT_ID"
+  automation_service_account_member = "REPLACE_AUTOMATION_SERVICE_ACCOUNT_MEMBER"
+}
+
+resource "google_project_service" "services" {
+  for_each = toset([
+    "iam.googleapis.com",
+    "iamcredentials.googleapis.com",
+    "sts.googleapis.com",
+  ])
+
+  project = local.project_id
+
+  service                    = each.value
+  disable_on_destroy         = false
+  disable_dependent_services = false
+}
+
+module "github_token_minter_github_wif" {
+  source = "git::https://github.com/abcxyz/github-token-minter.git//terraform/modules/github-wif?ref=c6c2a47034268b472dbf1e51c28118c5cedb61e4"
```

Comment:
@verbanicm In the design, I wrote that modules should be colocated within the source repo of the service, but having a copy of github-wif module in each service seems difficult in terms of maintenance/consistency. Should we have separate templates for common modules that can be rendered in the abcxyz-services repo?

Reply:
Combined into main github-token-minter module